README.md

```diff
@@ -18,20 +18,29 @@ For the latest changes, please read the [CHANGELOG](CHANGELOG.md).
 ![Quantum Circuit](./resources/quantum-circuit-examples/quantum-random-number.png)
 
 ## Table of Contents
-- [Prerequisites](#prerequisites)
-- [Installation](#installation)
-- [About Quantum Computing](#about-quantum-computing)
-  - [Qubit State](#qubit-state)
-  - [Qubit Measurement](#qubit-measurement)
-  - [Entanglement](#entanglement)
-- [Quantum Nodes](#quantum-nodes)
-- [Quantum Algorithm Nodes](#quantum-algorithm-nodes)
-- [Contributing](#contributing)
-- [Acknowledgements](#acknowledgements)
+- [node-red-quantum](#node-red-quantum)
+  - [Table of Contents](#table-of-contents)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+  - [About Quantum Computing](#about-quantum-computing)
+    - [Qubit State](#qubit-state)
+      - [Example:](#example)
+    - [Qubit Measurement](#qubit-measurement)
+      - [Example:](#example-1)
+    - [Entanglement](#entanglement)
+      - [Example:](#example-2)
+  - [Quantum Nodes](#quantum-nodes)
+    - [How to Use](#how-to-use)
+    - [Rules](#rules)
+  - [Quantum Algorithm Nodes](#quantum-algorithm-nodes)
+  - [Contributing](#contributing)
+  - [Acknowledgements](#acknowledgements)
```

Comment:
Usually the depth of headers in a table of contents shouldn't exceed H3, so I think H4 and beyond should be emitted from the TOC (in this case the "example" listings). Also I don't think it should include the "node-red-quantum" and "Table of Contents" listings, since the first wraps the entire TOC in an extra layer of indentation unnecessarily, whilst the second is redundant since the TOC doesn't need to include itself. What do you think?

```suggestion
- [Prerequisites](#prerequisites)
- [Installation](#installation)
- [About Quantum Computing](#about-quantum-computing)
  - [Qubit State](#qubit-state)
  - [Qubit Measurement](#qubit-measurement)
  - [Entanglement](#entanglement)
- [Quantum Nodes](#quantum-nodes)
  - [How to Use](#how-to-use)
  - [Rules](#rules)
- [Quantum Algorithm Nodes](#quantum-algorithm-nodes)
- [Contributing](#contributing)
- [Acknowledgements](#acknowledgements)
```

Reply:
Yeah that's also what I wanted to include in the TOC but using VS Code, the TOC automatically updates itself when I save the file. 

Sorry I did not notice the TOC changed.